client/lib/mixins/data-observe/README.md

```diff
@@ -1,18 +1,4 @@
 Data Change
 ===========
 
-A React mixin that makes it easy to trigger re-rendering of a component when a `prop` emits a `change` event.
-
-
-### Usage
-
-
-```js
-var observe = require( 'lib/mixins/data-observe' );
-
-var Component = React.createClass({
-	mixins: [ observe( 'sites', 'user' ) ]
-});
-```
-
-([Read more about React mixins](http://facebook.github.io/react/docs/reusable-components.html#mixins).)
+**This is deprecated! Please use Redux instead. Refer to the [docs on data handling](/docs/our-approach-to-data.md).**
```

Comment:
Even a deprecated class deserves some documentation, for those interested in what it does. Can we add the deprecation warning just as a prefix and leave the rest of the file intact?

Reply:
Amended the commit: dff5136